tests/test_diffusers.py

```diff
@@ -643,6 +643,9 @@ def test_no_generation_regression(self):
                 [0.70760196, 0.7136303, 0.7000798, 0.714934, 0.6776865, 0.6800843, 0.6923707, 0.6653969, 0.6408076]
             )
         image = outputs.images[0]
+        import matplotlib.pyplot as plt
+        plt.imsave('test_no_generation_regression_output.png', image)
```

Comment:
If `image` is a PIL image it could be `image.save("test_no_generation_regression_output.png")`.

Reply:
Also, if the outputs are pure latents (and are not decoded by the subsequent VAE in the process), then the output won't make to much of a sense.

Reply:
I agree with @sayakpaul. We can specify `output_type="pil"` when instantiating the pipeline and save the image as suggested.

Reply:
I had already tried the the conversion to PIL image like you mentioned @sayakpaul , but got a blank image instead. (Probably due to incorrect clamping)
@regisss , I am aware of the output_type="pil" but did not use that since the test is checking for raw pixel values later.

Will be trying out this diffusers numpy_to_pil function and update: https://github.com/huggingface/diffusers/blob/v0.23.1/src/diffusers/image_processor.py#L82